src/androidTest/java/com/mixpanel/android/mpmetrics/AutomaticEventsTest.java

```diff
@@ -218,13 +217,13 @@ public void testAutomaticMultipleInstances() throws InterruptedException {
         int initialCalls = 2;
         mLatch = new CountDownLatch(initialCalls);
         final CountDownLatch secondLatch = new CountDownLatch(initialCalls);
-        final BlockingQueue<String> secondPerformedRequests =  new LinkedBlockingQueue<>();
+        final BlockingQueue<String> secondPerformedRequests = new LinkedBlockingQueue<>();
 
         final HttpService mpSecondPoster = new HttpService() {
             @Override
-            public byte[] performRequest(String endpointUrl, Map<String, Object> params, SSLSocketFactory socketFactory) throws ServiceUnavailableException, IOException {
```

Comment:
what's the reason for removing the exception throws?

Reply:
cause they are never thrown
![image](https://user-images.githubusercontent.com/36679208/116756591-38b37780-a9c1-11eb-93b7-bbf7190ac26c.png)